src/batou_ext/nixos.py

```diff
@@ -0,0 +1,85 @@
+from pathlib import Path
+
+from batou.component import Component
+from batou.lib.file import Purge
+
+from batou_ext.nix import NixFile, component_to_nix, nix_dict_to_nix
+
+# XXX: error messages stemming from the "batouModule" are displayed with
+# wrong location information, it shows the glue module instead of the actual
+# module. Can we fix that somehow?
```

Comment:
Just played around with this and this doesn't seem to be the case anymore (to be more specific: the `imports` is the reason for that, when exposing `batouModule` directly in the `in`-section, this wouldn't work).

The only case where the glue code is shown in the error is when having an invalid binding in the function argument, i.e.:

```nix
{ config, lib, pkgs, component, this_doesnt_exist, ... }:

{ /* ... */ }
```

Did I miss a case @dpausp @zagy? Otherwise I'm happy to file a PR to remove that :)